### DISCUSSION TITLE
Legal blocker: Cannot use Discussions feature for customer-support use-cases

### DISCUSSION THREAD

**Discussion by petemounce** [community/community#19976] [Discussions]
I'd _like_ to use github discussions and github issues/projects for customer-support use-cases. For example
* customer product feedback, bug reports, etc
* customer support - product questions, etc

However, I haven't figured out a way past one blocker that I have.

We have a requirement (since we and customers sign mutual NDAs in some cases) that some of our customers remain customers-in-secret - it's a contractual obligation that our relationship not leak.

So this implies a discussion forum would need to be private+secret to just that customer and us
* RULED OUT [org-level discussion forums](https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization) because 
    * looks like no way to make segments only visible to certain groups of org-members
* so, one repository per customer; invite customer-employees to organisation; add to `customer-{name}` team; grant team as `maintainer` on customer repo (we want them to be able to manage issues as well as ourselves) (or run [`prow`](https://github.com/kubernetes/test-infra/blob/master/prow/README.md#functions-and-features) for an IssueOps workflow)
    * BUT now each customer-employee can see every other customer-employee
        * ... because organisation-membership cannot be `secret`, only `public|private`.
    * BUT now we're paying a client-access license per customer-employee
        * ... which we'd like to avoid
            * since customer-employees will really only be using discussions and _some_ issues features
            * and customer-employees increases our spend `O(customers * participating-customer-employee)` on top of our own `O(employees)`.

Are there capabilities or docs I've missed? At present, I think this blocks our usage of github discussions. (I reckon we're not unique in the mutual-NDA setup that we do.)

**Comment by seanlabastille** [comment]
I am currently addressing a similar challenge.
The best fit so far seems to be: a separate organisation dedicated to interactions with customers or integrators, private teams and team discussions.

Using a separate organisation allows you to benefit from free pricing if you don't need any paid features.

However: a limitation of team discussions seem to be that they don't have the same feature set as repository/organisation discussions and also don't trigger web hooks or other automations which would be helpful to keep track of customer feedback in other systems.
